test/e2e/data/infrastructure-docker/main/cluster-template-dualstack/cluster-dualstack.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
nit: you don't need the -dualstack suffixes as that is already part of the folder name

But up to you